samples/VanillaJSTestApp2.0/app/multiple_resources/test/multiple_resources.spec.ts

```diff
@@ -0,0 +1,115 @@
+import "mocha";
+import puppeteer from "puppeteer";
+import { expect } from "chai";
+import fs from "fs";
```

Comment:
I would recommend using the promise fs APIs: https://nodejs.org/api/fs.html#fs_fs_promises_api

```js
import fs from "fs/promises";
```